tests/Issues/Issue7099/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Rector\Core\Tests\Issues\Issue7099\Fixture;
+
+class Document {
+
+}
+
+class View {
+    private $data;
+
+    public function __construct($data) {
+        $this->data = $data;
+    }
+
+    /** @return mixed */
+    public function getData() {
+        return $this->data;
+    }
+}
```

Comment:
The classes that are not being changed, but are only used to analyze, should be in own `/Source` directory. With correct namespace and PSR-4 autoloaded.

E.g. https://github.com/rectorphp/rector-src/tree/main/rules-tests/Renaming/Rector/PropertyFetch/RenamePropertyRector/Source